bi-connectors/PowerBIConnector/SqlOdbcPBIConnector.pq

```diff
@@ -56,9 +66,29 @@ SqlOdbcPBIConnectorImpl = (Server as text) as table =>
                     UseSSL = 0
                 ],
 
+        // Substract the server from the user input in case if it entered like 'http://localhost' or 'https://srv.com:100500' or 'localhost:0'
+        ServerWithoutPrefix =
```

Comment:
Have you thought about using https://docs.microsoft.com/en-us/powerquery-m/uri-parts instead which would simplify the below?

Reply:
Thank you for the idea! Going to fix.

Reply:
Fixed in 26d29ac